.github/main.workflow

```diff
@@ -1,22 +1,28 @@
 workflow "Create Release" {
   on = "push"
-  resolves = ["Release"]
-}
-
-action "If Master" {
-  uses = "actions/bin/filter@3c0b4f0e63ea54ea5df2914b4fabf383368cd0da"
-  args = "branch master"
+  resolves = ["Release", "Lint"]
 }
 
 action "Install Dependencies" {
   uses = "actions/npm@59b64a598378f31e49cb76f27d6f3312b582f680"
   args = "install"
-  needs = ["If Master"]
 }
 
-action "Build" {
+action "Lint" {
   uses = "actions/npm@59b64a598378f31e49cb76f27d6f3312b582f680"
+  args = "run lint:ci"
   needs = ["Install Dependencies"]
+}
+
+action "If Master" {
+  uses = "actions/bin/filter@3c0b4f0e63ea54ea5df2914b4fabf383368cd0da"
+  args = "branch master"
+  needs = ["Lint"]
```

Comment:
this seems weird, i feel like this filter should be left alone? Workflow already runs lint so it would fail if linting fails right. everything else makes sense to me though. Even if you had Build need Lint that would make more sense to me. is this cause you dont want build to run until lint is done?